Removed bad font from javadoc stylesheet

astrelsky opened the pull request:
The new stylesheet is exactly the same as the default stylesheet
with the exception of the Georgia font being replaced with Arial.
Resolves #1576

astrelsky commented:
@ryanmkurtz by any chance do you know of a better solution?

ryanmkurtz commented:
The only other option I can think of is to edit the stylesheet it produces afterwards.  You could probably do it in just a couple of gradle lines, making for a much simpler pull request.

astrelsky commented:
> The only other option I can think of is to edit the stylesheet it produces afterwards. You could probably do it in just a couple of gradle lines, making for a much simpler pull request.

Should such changes go in the zipJavadocs task?

ryanmkurtz commented:
I think so.  Gradle changes were always trial and error for me.  There are doFirst and doLast expressions you can use to control when things happen.  Maybe in a doFirst in zipJavadocs?  Or a doLast in createJavadocs?  You can also add a filter to a copy task where you might be able to strip out a font somehow.  An example of that is in extensionCommon.gradle.

astrelsky commented:
> I think so. Gradle changes were always trial and error for me. There are doFirst and doLast expressions you can use to control when things happen. Maybe in a doFirst in zipJavadocs? Or a doLast in createJavadocs? You can also add a filter to a copy task where you might be able to strip out a font somehow. An example of that is in extensionCommon.gradle.

Thank you. I'll probably revise this to take that approach over the weekend.

astrelsky commented:
I have changed the gradle script so that the Georgia font is now replaced with Arial in the stylesheet after the javadocs are built. If there is a default font preferred over Arial I'll can set it to that instead.

ryanmkurtz commented:
Nice clean solution!

ryanmkurtz commented:
After revisiting this, it doesn't seem worth the extra gradle hackery and maintenance to alter the font.  Maybe if more complaints about it popped up we could reconsider it.
